nix/pkgs/cardano-ledger-release-tool.nix

```diff
@@ -1,24 +1,8 @@
-{
-  stdenv,
-  lib,
-  fetchzip,
-}:
-# Although `cardano-ledger-release-tool` is a flake, we choose to download
-# the released binary artifact for compatibility with the non-nix CI
+{ system }:
 let
-  pname = "cardano-ledger-release-tool";
-  version = "0.1.0.0";
-  hash = "sha256-Xge1/T5mtsOF5XiIDPoWU2f2HMJ0pCEbS+brfQyCZpw=";
-
-  repo = "input-output-hk/${pname}";
-  release = "${pname}-${version}";
-  url = "https://github.com/${repo}/releases/download/${release}/${pname}-Linux-x86_64.zip";
+  type = "github";
+  owner = "input-output-hk";
+  repo = "cardano-ledger-release-tool";
+  rev = "f52262644af02e6eae54cf5d6021ecebf7f5c1d1";
```

Comment:
Note that it's not possible to use a tag here, because nix considers that an "unlocked" input since tags can be changed whereas commit hashes are content-based

Reply:
Conversely, it's not possible to use a commit hash in the [GitHub workflow](https://github.com/IntersectMBO/cardano-ledger/blob/master/.github/workflows/haskell.yml#L659) because the static binary has to be downloaded from a release artifact.